README.md

```diff
@@ -19,6 +19,14 @@ kubectl apply -f https://raw.githubusercontent.com/FoundationDB/fdb-kubernetes-o
 kubectl apply -f https://raw.githubusercontent.com/foundationdb/fdb-kubernetes-operator/master/config/samples/deployment.yaml
 ```
 
+Or by installing the helm chart (*see values.yaml [here](charts/fdb-operator/values.yaml)*):
+
+```bash
+helm repo add fdb-kubernetes-operator https://foundationdb.github.io/fdb-kubernetes-operator/
```

Comment:
This won't work at the time this is merged, because we don't have the GitHub pages configured. I'd rather we track that as a follow-up issue, and defer any links to those pages until they are live.